_layouts/tutorial.html

```diff
@@ -64,6 +64,36 @@ <h3 class="title">Code example:</h3>
             Basic Kafka
           </option>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.scala == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/scala.html" {% if page.stack == 'scala' %} selected {% endif %}>
+          Scala
+          </option>
```

Comment:
The same comment as below, only add `scala` and update when we add additional tutorials.